### DISCUSSION TITLE
Initializing the wind direction and speed

### DISCUSSION THREAD

**Discussion by RobertXWL** [JSBSim-Team/jsbsim#778] [Q&A]
Hello All JSBsim members,
         I'm trying to initialize the wind for the environment like this
         "initial_windx = Property('ic/vw-bx-fps', 'Wind velocity initial condition in Body X frame in feet/second [rad/s]')
         initial_windy = Property('ic/vw-by-fps', 'Wind velocity initial condition in Body Y frame in feet/second [rad/s]')
         initial_windz = Property('ic/vw-bz-fps', 'Wind velocity initial condition in Body Z frame in feet/second [rad/s]')"
         but I failed
         please explain for me how can I initialize the wind for the environment, or change the wind while flying, without using the script, 
         and how should I define the properties of the initial wind
         Thank you very much

**Comment by bcoconni** [comment]
@RobertXWL, the properties you need to modify to change the wind heading and velocity are described in discussion #764:
> Wind can be modified using the properties `atmosphere/psiw-rad` (wind direction) and `atmosphere/wind-mag-fps` (wind velocity).
